mountpoint-s3/tests/fuse_tests/fork_test.rs

```diff
@@ -912,10 +906,9 @@ fn get_mount_from_source_and_mountpoint(source: &str, mount_point: &str) -> Opti
     let mut cmd = Command::new("mount");
     #[cfg(target_os = "linux")]
     cmd.arg("-l");
-    let mut cmd = cmd.stdout(Stdio::piped()).spawn().expect("Unable to spawn mount tool");
```

Comment:
We were not waiting for spawned command here, and Clippy was asking to add `.wait`. I've changed it to use `.output` instead which does wait and returns the output of the command.